add-new-exercise.ps1

```diff
@@ -0,0 +1,54 @@
+param (
```

Comment:
I think the default PowerShell convention is to use PascalCasing for parameters.

Reply:
fixed